src/utils.js

```diff
@@ -398,10 +398,11 @@ function firstStableVersion(supportInfo) {
     if (
       !Object.prototype.hasOwnProperty.call(supportEntry, 'flags') &&
       (!versionAdded ||
-        !basicCompatVersionComparison(
-          supportEntry.version_added,
-          parseInt(versionAdded, 10)
-        ))
+        (supportEntry.version_added &&
+          !basicCompatVersionComparison(
+            supportEntry.version_added,
+            parseInt(versionAdded, 10)
+          )))
```

Comment:
> we may actually return false if versionAdded is `false`, to make it handle the `versionAdded: false` case explicitly (at the moment I think we do just return true if `versionAdded` isn't a string, which I think it is ok because we are only calling it for `versionAdded: true` or `versionAdded: "NN"`, with "NN" I mean a string that parse as a number).
> 
> @willdurand what do you think?

Looks like it is covered by a test case already: https://github.com/mozilla/addons-linter/blob/renovate/mdn-browser-compat-data-3.x/tests/unit/test.utils.js#L516

Reply:
great, I missed to spot it, even better!